component/vshn_postgres.jsonnet

```diff
@@ -776,6 +778,65 @@ local prometheusRule = {
                     'for': '120m',
                     labels: {
                       severity: 'critical',
+                      syn_team: 'schedar',
+                    },
+                  },
+                ],
+              },
+              // new
+              {
+                name: 'postgresql-replication',
+                rules: [
+                  {
+                    alert: 'PostgreSQLPodReplicasCritical',
```

Comment:
I mentioned in the chat already, but I previously commented here on the wrong line 🙈 

The name for that one was good, but the one on line 829 should be called `PostgreSQLPodReplicasCritical`.

Reply:
ok, restored that